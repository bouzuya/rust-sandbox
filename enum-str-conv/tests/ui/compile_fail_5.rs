fn main() {
    #[derive(enum_str_conv::EnumStrConv)]
    // No unknown fn is provided
    #[enum_str_conv(error = ())]
    enum MyEnum {
        #[enum_str_conv(str = "apple")]
        A,
    }
}
