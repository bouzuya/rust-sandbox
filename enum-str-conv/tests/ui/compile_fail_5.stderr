error: expected `unknown` argument: #[enum_str_conv(unknown = ...)]
 --> tests/ui/compile_fail_5.rs:4:7
  |
4 |     #[enum_str_conv(error = ())]
  |       ^^^^^^^^^^^^^^^^^^^^^^^^^
